### PR TITLE
Code Quality: Refactor LayoutPages & ShellPages

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Motivation and Context

- [x] Add comments in LayoutPages & ShellPages XAML files.
- [x] Refactor above files' code-behind

### PR Checklist
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

_No Response_

### CODE REVIEW

File: src/Files.App/App.xaml.cs [hunk 1]
```diff
@@ -439,9 +459,7 @@ private static void AppUnhandledException(Exception ex, bool shouldShowNotificat
 		}
 
 		public static void CloseApp()
-		{
-			Window.Close();
-		}
+			=> Window.Close();
```
Comment (yaira2) [inline src/Files.App/App.xaml.cs:462]: @d2dyno1 are we supposed to use brackets here?

File: src/Files.App/BaseLayout.cs [hunk 2]
```diff
@@ -514,9 +514,7 @@ public void SetSelectedItemsOnNavigation()
 					ItemManipulationModel.FocusFileList();
 				}
 			}
-			catch (Exception)
-			{
-			}
+			catch (Exception) { }
```
Comment (yaira2) [inline src/Files.App/BaseLayout.cs:517]: And here?

File: src/Files.App/IShellPage.cs [hunk 3]
```diff
@@ -94,9 +94,9 @@ public interface IPaneHolder : IDisposable, INotifyPropertyChanged
 
 	public interface IMultiPaneInfo
 	{
-		// The instance is the left (or only) pane
+		// The instance is the left or only one pane
```
Comment (yaira2) [inline src/Files.App/IShellPage.cs:97]:
```suggestion
		// The instance is the left or only pane
```

File: src/Files.App/MainWindow.xaml.cs [hunk 4]
```diff
@@ -66,7 +60,7 @@ public async Task InitializeApplication(object activatedEventArgs)
 			var rootFrame = EnsureWindowIsInitialized();
 			Activate();
 
-			// WINUI3: port activation args from App.xaml.cs.old: OnActivated, OnFileActivated
+			// WINUI3: Port activation args from App.xaml.cs.old: OnActivated, OnFileActivated
```
Comment (yaira2) [inline src/Files.App/MainWindow.xaml.cs:63]: Can we remove this comment now?

File: src/Files.App/Views/BaseShellPage.cs [hunk 5]
```diff
@@ -210,12 +214,11 @@ public BaseShellPage(CurrentInstanceViewModel instanceViewModel)
...
-			TODO UA307 Default back button in the title bar does not exist in WinUI3 apps.
-			The tool has generated a custom back button in the MainWindow.xaml.cs file.
-			Feel free to edit its position, behavior and use the custom back button instead.
-			Read: https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/case-study-1#restoring-back-button-functionality
-			*/
+			// TODO:
+			//  UA307 Default back button in the title bar does not exist in WinUI3 apps.
+			//  The tool has generated a custom back button in the MainWindow.xaml.cs file.
+			//  Feel free to edit its position, behavior and use the custom back button instead.
+			//  Read: https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/case-study-1#restoring-back-button-functionality
```
Comment (yaira2) [inline src/Files.App/Views/BaseShellPage.cs:221]: Can this be removed now?

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml [hunk 6]
```diff
@@ -120,15 +121,19 @@
...
 	<Grid x:Name="RootGrid" ContextFlyout="{x:Bind BaseContextMenuFlyout}">
+
+		<!--  Empty Indicator  -->
 		<uc:FolderEmptyIndicator
 			HorizontalAlignment="Center"
 			VerticalAlignment="Center"
 			Canvas.ZIndex="0"
 			EmptyTextType="{x:Bind ParentShellPageInstance.FilesystemViewModel.EmptyTextType, Mode=OneWay}" />
 
+		<!--  Unable To Use For Item Name Tip  -->
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/ColumnViewBase.xaml:136]:
```suggestion
		<!--  Invalid Item Name Tip  -->
```

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml [hunk 7]
```diff
@@ -382,26 +400,34 @@
 					<ListView.ItemTemplate>
 						<DataTemplate x:DataType="ICollectionViewGroup">
 							<StackPanel
-								Margin="0,4,0,4"
+								Margin="0,4"
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/ColumnViewBase.xaml:403]: Can you revert? Coding convention is to explicitly define the margin. The same goes for grid rows (we always define `0` even though it's the default.

File: src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml [hunk 8]
```diff
@@ -8,7 +8,6 @@
 	xmlns:icore="using:Microsoft.Xaml.Interactions.Core"
 	xmlns:local="using:Files.App"
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	x:Name="PageRoot"
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml:11]: Are you sure this isn't being used in an item template?

File: src/Files.App/Views/LayoutModes/StandardLayoutMode.cs [hunk 9]
```diff
@@ -276,11 +272,8 @@ protected async void RenameTextBox_KeyDown(object sender, KeyRoutedEventArgs e)
 						NextRenameIndex = 0;
 						EndRename(textBox);
 
-						if
-						(
-							newIndex >= 0 &&
-							newIndex < ListViewBase.Items.Count
-						)
+						if (newIndex >= 0 &&
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/StandardLayoutMode.cs:275]: Parenthesis should be on their own line

File: src/Files.App/Views/LayoutModes/StandardLayoutMode.cs [hunk 10]
```diff
@@ -297,11 +290,8 @@ protected bool TryStartRenameNextItem(ListedItem item)
 			var nextItemIndex = ListViewBase.Items.IndexOf(item) + NextRenameIndex;
 			NextRenameIndex = 0;
 
-			if
-			(
-				nextItemIndex >= 0 &&
-				nextItemIndex < ListViewBase.Items.Count
-			)
+			if (nextItemIndex >= 0 &&
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/StandardLayoutMode.cs:293]: Same here

File: src/Files.App/Views/MainPage.xaml.cs [hunk 11]
```diff
@@ -32,26 +32,21 @@
...
 		public SidebarViewModel SidebarAdaptiveViewModel = Ioc.Default.GetRequiredService<SidebarViewModel>();
 
 		public AppModel AppModel => App.AppModel;
 
-		public MainPageViewModel ViewModel
-		{
-			get => (MainPageViewModel)DataContext;
-			set => DataContext = value;
-		}
+		public MainPageViewModel ViewModel { get; set; }
```
Comment (yaira2) [inline src/Files.App/Views/MainPage.xaml.cs:49]: Can you use dependency injection for this? Normally I'd say it's out of scope for the PR but it might be a good idea to take care of it while you're in here.

File: src/Files.App/Views/PaneHolderPage.xaml [hunk 12]
```diff
@@ -34,14 +33,38 @@
...
+			Invoked="KeyboardAccelerator_Invoked"
+			Modifiers="Menu,Shift" />
+	</Page.KeyboardAccelerators>
+
+	<Grid x:Name="RootGrid">
 		<Grid.ColumnDefinitions>
 			<ColumnDefinition
 				x:Name="LeftColumn"
-				Width="1.0*"
+				Width="*"
```
Comment (yaira2) [inline src/Files.App/Views/PaneHolderPage.xaml:67]: Why the change?

File: src/Files.App/Views/PaneHolderPage.xaml.cs [hunk 13]
```diff
@@ -74,13 +76,9 @@ private bool windowIsCompact
 
 		public bool IsMultiPaneActive => IsRightPaneVisible;
 
-		public bool IsMultiPaneEnabled
-		{
-			get => !(App.Window.Bounds.Width <= 750);
-		}
+		public bool IsMultiPaneEnabled => !(App.Window.Bounds.Width <= 750);
```
Comment (yaira2) [inline src/Files.App/Views/PaneHolderPage.xaml.cs:79]: Can you add back the brackets?

File: src/Files.App/Views/WidgetsPage.xaml.cs [hunk 14]
```diff
@@ -31,11 +32,7 @@ public sealed partial class WidgetsPage : Page, IDisposable
 		private FileTagsWidget fileTagsWidget;
 		private RecentFilesWidget recentFilesWidget;
 
-		public YourHomeViewModel ViewModel
-		{
-			get => (YourHomeViewModel)DataContext;
-			set => DataContext = value;
-		}
+		public YourHomeViewModel ViewModel { get; set; }
```
Comment (yaira2) [inline src/Files.App/Views/WidgetsPage.xaml.cs:35]: Can you use dependency injection for this as well?

File: src/Files.App/Views/WidgetsPage.xaml.cs [hunk 15]
```diff
@@ -61,15 +59,21 @@ private void ViewModel_WidgetListRefreshRequestedInvoked(object? sender, EventAr
 
 		public void ReloadWidgets()
 		{
-			quickAccessWidget = WidgetsHelpers.TryGetWidget<QuickAccessWidget>(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadQuickAccessWidget, quickAccessWidget);
-			drivesWidget = WidgetsHelpers.TryGetWidget<DrivesWidget>(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadDrivesWidget, drivesWidget);
-			bundlesWidget = WidgetsHelpers.TryGetWidget<BundlesWidget>(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadBundles, bundlesWidget);
-			fileTagsWidget = WidgetsHelpers.TryGetWidget<FileTagsWidget>(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadFileTags, fileTagsWidget);
-			recentFilesWidget = WidgetsHelpers.TryGetWidget<RecentFilesWidget>(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadRecentFiles, recentFilesWidget);
+			quickAccessWidget = WidgetsHelpers.TryGetWidget(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadQuickAccessWidget, quickAccessWidget);
```
Comment (yaira2) [inline src/Files.App/Views/WidgetsPage.xaml.cs:62]: @d2dyno1 are you okay with this?

File: src/Files.App/MainWindow.xaml.cs (hunk 4 above)
Comment (0x5bfa) [reply src/Files.App/MainWindow.xaml.cs:63]: Certainly

File: src/Files.App/Views/BaseShellPage.cs (hunk 5 above)
Comment (0x5bfa) [reply src/Files.App/Views/BaseShellPage.cs:221]: Also certainly

File: src/Files.App/Views/PaneHolderPage.xaml (hunk 12 above)
Comment (0x5bfa) [reply src/Files.App/Views/PaneHolderPage.xaml:67]: 1*, 1.0*, *

All of them is the same and should be constant.

File: src/Files.App/Views/WidgetsPage.xaml.cs (hunk 14 above)
Comment (0x5bfa) [reply src/Files.App/Views/WidgetsPage.xaml.cs:35]: Is it now, this place?

File: src/Files.App/Views/WidgetsPage.xaml.cs (hunk 15 above)
Comment (0x5bfa) [reply src/Files.App/Views/WidgetsPage.xaml.cs:62]: FYI, certainly sure.

```cs
public static TWidget? TryGetWidget<TWidget>(
    IPreferencesSettingsService preferencesSettingsService,
    WidgetsListControlViewModel widgetsViewModel,
    out bool shouldReload, TWidget? defaultValue = default)
        where TWidget : IWidgetItemModel, new()
```

Adding target type like TryGetWidget<TWidget>() is redundant because type will be discovered by the last parameter `TWidget? defaultValue = default`

File: src/Files.App/Views/PaneHolderPage.xaml.cs (hunk 13 above)
Comment (0x5bfa) [reply src/Files.App/Views/PaneHolderPage.xaml.cs:79]: Yeah, can you show the guild-line for this?

File: src/Files.App/Views/LayoutModes/StandardLayoutMode.cs (hunk 9 above)
Comment (0x5bfa) [reply src/Files.App/Views/LayoutModes/StandardLayoutMode.cs:275]: Oh, why on earth should be on? This is pretty old writing style of C language and I'm quite sure shouldn't be like this.

File: src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml (hunk 8 above)
Comment (0x5bfa) [reply src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml:11]: I'm sorry. Fixed.

File: src/Files.App/Views/PaneHolderPage.xaml.cs (hunk 13 above)
Comment (0x5bfa) [reply src/Files.App/Views/PaneHolderPage.xaml.cs:79]: set is not defined. so I think we can leave as it is.

File: src/Files.App/Views/LayoutModes/StandardLayoutMode.cs (hunk 9 above)
Comment (yaira2) [reply src/Files.App/Views/LayoutModes/StandardLayoutMode.cs:275]: I was thinking we'd want to do the same as we do for brackets.

File: src/Files.App/Views/PaneHolderPage.xaml.cs (hunk 13 above)
Comment (yaira2) [reply src/Files.App/Views/PaneHolderPage.xaml.cs:79]: I find having it on a separate line is more readable

File: src/Files.App/Views/WidgetsPage.xaml.cs (hunk 14 above)
Comment (0x5bfa) [reply src/Files.App/Views/WidgetsPage.xaml.cs:35]: YourHomeViewModel's constructor has two params so can't.

File: src/Files.App/Views/LayoutModes/StandardLayoutMode.cs (hunk 9 above)
Comment (0x5bfa) [reply src/Files.App/Views/LayoutModes/StandardLayoutMode.cs:275]: I think waste of lines...

File: src/Files.App/MainWindow.xaml [hunk 16]
```diff
@@ -7,9 +7,9 @@
 	xmlns:winuiex="using:WinUIEx"
 	mc:Ignorable="d">
 
+	<!--  Mica Alt (winui3gallery://item/SystemBackdrops)  -->
```
Comment (yaira2) [inline src/Files.App/MainWindow.xaml:10]: We do we link to the WinUI3 gallery?

File: src/Files.App/Views/LayoutModes/StandardLayoutMode.cs (hunk 9 above)
Comment (yaira2) [reply src/Files.App/Views/LayoutModes/StandardLayoutMode.cs:275]: 👍

File: src/Files.App/Views/PaneHolderPage.xaml.cs (hunk 13 above)
Comment (0x5bfa) [reply src/Files.App/Views/PaneHolderPage.xaml.cs:79]: Hoe about being like this?

```cs
public bool IsMultiPaneEnabled
     => !(App.Window.Bounds.Width <= 750);
```

File: src/Files.App/Views/PaneHolderPage.xaml.cs [hunk 17]
```diff
@@ -205,7 +195,7 @@ public PaneHolderPage()
 			ActivePane = PaneLeft;
 			IsRightPaneVisible = IsMultiPaneEnabled && UserSettingsService.PreferencesSettingsService.AlwaysOpenDualPaneInNewTab;
 
-			// TODO: fallback / error when failed to get NavigationViewCompactPaneLength value?
+			// TODO?: Fallback or occur an error when failed to get NavigationViewCompactPaneLength value
```
Comment (yaira2) [inline src/Files.App/Views/PaneHolderPage.xaml.cs:198]:
```suggestion
			// TODO?: Fallback or an error can occur when failing to get NavigationViewCompactPaneLength value
```

File: src/Files.App/Views/ModernShellPage.xaml.cs [hunk 18]
```diff
@@ -356,21 +366,21 @@ public override void NavigateToPath(string? navigationPath, Type? sourcePageType
 
 				NavigationTransitionInfo transition = new SuppressNavigationTransitionInfo();
 
-				if (sourcePageType == typeof(WidgetsPage)
-					|| ItemDisplayFrame.Content.GetType() == typeof(WidgetsPage) &&
+				if (sourcePageType == typeof(HomePage) ||
```
Comment (yaira2) [inline src/Files.App/Views/ModernShellPage.xaml.cs:369]: Why was the page renamed?

File: src/Files.App/Views/MainPage.xaml.cs [hunk 19]
```diff
@@ -312,32 +304,38 @@ private void SidebarControl_SidebarItemInvoked(object sender, SidebarItemInvoked
...
-						else if (ItemPath.Equals("Home", StringComparison.OrdinalIgnoreCase)) // Home item
+						// Home item
+						else if (ItemPath.Equals("Home", StringComparison.OrdinalIgnoreCase))
 						{
 							if (ItemPath.Equals(SidebarAdaptiveViewModel.SidebarSelectedItem?.Path, StringComparison.OrdinalIgnoreCase))
 								return; // return if already selected
 
 							navigationPath = "Home";
-							sourcePageType = typeof(WidgetsPage);
+							sourcePageType = typeof(HomePage);
```
Comment (yaira2) [inline src/Files.App/Views/MainPage.xaml.cs:332]: What's going on here?

File: src/Files.App/Views/MainPage.xaml.cs [hunk 20]
```diff
@@ -60,14 +54,13 @@ public MainPageViewModel ViewModel
...
 		private bool keyReleased = true;
 
-		public readonly OngoingTasksViewModel OngoingTasksViewModel;
+		public readonly OngoingTasksViewModel OngoingTasksViewModel = Ioc.Default.GetRequiredService<OngoingTasksViewModel>();
 
 		public MainPage()
 		{
 			InitializeComponent();
-			DataContext = Ioc.Default.GetRequiredService<MainPageViewModel>();
-			OngoingTasksViewModel = Ioc.Default.GetRequiredService<OngoingTasksViewModel>();
```
Comment (yaira2) [inline src/Files.App/Views/MainPage.xaml.cs:69]: Can you use dependency injection (building off of @lukeblevins work)?

File: src/Files.App/Views/MainPage.xaml.cs [hunk 21]
```diff
@@ -32,26 +33,19 @@
...
 
-		public AppModel AppModel => App.AppModel;
+		public static AppModel AppModel => App.AppModel;
 
-		public MainPageViewModel ViewModel
-		{
-			get => (MainPageViewModel)DataContext;
-			set => DataContext = value;
-		}
+		public MainPageViewModel ViewModel { get; } = Ioc.Default.GetRequiredService<MainPageViewModel>();
```
Comment (yaira2) [inline src/Files.App/Views/MainPage.xaml.cs:48]: Dependency injection should be used here as well

Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml.cs:48]: I did already. What do you mean? You mean we should do this in the constructor?

File: src/Files.App/Views/MainPage.xaml [hunk 22]
```diff
@@ -150,7 +156,6 @@
 								VerticalAlignment="Bottom"
 								AutomationProperties.Name="{x:Bind Commands.NewTab.AutomationName}"
 								Background="Transparent"
-								BorderBrush="Transparent"
```
Comment (yaira2) [inline src/Files.App/Views/MainPage.xaml:153]: I'm pretty sure this needs to be defined, please revert for now.

File: src/Files.App/Views/MainPage.xaml.cs (hunk 20 above)
Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml.cs:69]: I did define & assign at the same time.

File: src/Files.App/Views/MainPage.xaml (hunk 22 above)
Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml:153]: BorderThinkess is already set 0 and so there's no effect. In spite of that, should?

File: src/Files.App/Views/MainPage.xaml.cs (hunk 21 above)
Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml.cs:48]: The preferred way is to register the view model as a singleton in app.xaml.cs and assign it through the constructor.

File: src/Files.App/Views/PaneHolderPage.xaml.cs (hunk 13 above)
Comment (yaira2) [reply src/Files.App/Views/PaneHolderPage.xaml.cs:79]: That's certainly better

File: src/Files.App/Views/ModernShellPage.xaml.cs (hunk 18 above)
Comment (0x5bfa) [reply src/Files.App/Views/ModernShellPage.xaml.cs:369]: **Before**
- ViewModel: YourHomeViewModel
- Page: WidgetsPage

**After**
- ViewModel: HomeViewModel
- Page: HomePage

**Validation**
- This page is used as Home page
- Should not limit the page scope to Widgets

File: src/Files.App/Views/MainPage.xaml (hunk 22 above)
Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml:153]: Is there a VisualState? I remember it was here for a reason but it's possible that it no longer applies.

File: src/Files.App/Views/ModernShellPage.xaml.cs (hunk 18 above)
Comment (yaira2) [reply src/Files.App/Views/ModernShellPage.xaml.cs:369]: > This page is used as Home page

It used to be called HomePage but it was repurposed as Widgets, if anything the ViewModel should be renamed to WidgetsViewModel. The reasoning behind this is that we might allow users to change the home page in the future.

Comment (0x5bfa) [reply src/Files.App/Views/ModernShellPage.xaml.cs:369]: Understood.

File: src/Files.App/Views/MainPage.xaml (hunk 22 above)
Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml:153]: There isn't. Totally validated.

Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml:153]: Alright then, thanks for checking!

File: src/Files.App/Views/MainPage.xaml.cs (hunk 21 above)
Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml.cs:48]: As a lot of code was written like this, I thought this way was preferred. Doesn't matter, I'll fix.

Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml.cs:48]: See #11820 for reference.

Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml.cs:48]: > As a lot of code was written like this, I thought this way was preferred. Doesn't matter, I'll fix.

It's a very recent change 👍

File: src/Files.App/MainWindow.xaml (hunk 16 above)
Comment (0x5bfa) [reply src/Files.App/MainWindow.xaml:10]: Removed

File: src/Files.App/Views/MainPage.xaml.cs (hunk 20 above)
Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml.cs:69]: Fixed like this in the constructor.

```cs
// Dependency Injection
UserSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
Commands = Ioc.Default.GetRequiredService<ICommandManager>();
WindowContext = Ioc.Default.GetRequiredService<IWindowContext>();
SidebarAdaptiveViewModel = Ioc.Default.GetRequiredService<SidebarViewModel>();
ViewModel = Ioc.Default.GetRequiredService<MainPageViewModel>();
OngoingTasksViewModel = Ioc.Default.GetRequiredService<OngoingTasksViewModel>();
```

File: src/Files.App/Views/MainPage.xaml.cs (hunk 21 above)
Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml.cs:48]: Fixed.

File: src/Files.App/Views/PaneHolderPage.xaml.cs (hunk 13 above)
Comment (0x5bfa) [reply src/Files.App/Views/PaneHolderPage.xaml.cs:79]: Fixed

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml [hunk 23]
```diff
@@ -176,14 +184,11 @@
...
-						<DataTemplate x:DataType="local2:ListedItem">
+						<DataTemplate x:DataType="filesystem:ListedItem">
 							<Grid
 								x:Name="FilesRootGrid"
-								Height="30"
-								Margin="0"
-								Padding="12,0,12,0"
-								HorizontalAlignment="Stretch"
-								VerticalAlignment="Center"
+								Height="32"
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/ColumnViewBase.xaml:190]: Height should be set to `30` in this case.

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml [hunk 24]
```diff
@@ -176,14 +184,11 @@
...
+								Padding="12,0"
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/ColumnViewBase.xaml:191]:
```suggestion
								Padding="12,0,12,0"
```
Padding should be explicitly defined.

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml (hunk 23 above)
Comment (0x5bfa) [reply src/Files.App/Views/LayoutModes/ColumnViewBase.xaml:190]: Fixed

File: src/Files.App/Views/MainPage.xaml.cs (hunk 20 above)
Comment (lukeblevins) [reply src/Files.App/Views/MainPage.xaml.cs:69]: This should suffice until we have a proper navigation service.

**Review by yaira2 — APPROVED** [review]
LGTM
